app/controllers/concerns/course/users_controller_management_concern.rb

```diff
@@ -49,7 +51,10 @@ def requests # :nodoc:
   end
 
   def invitations # :nodoc:
-    @course_users = @course_users.joins { invitation }.includes(invitation: :user_email)
+    @course_users = @course_users.joins { invitation }.includes(invitation: :user_email).
+                    order_alphabetically
+    @course_users = @course_users.partition { |user| user.workflow_state == 'approved' }
```

Comment:
It does work.

Would the other states `rejected` and `requested` show up on this page?